cg/models/nf_analysis.py

```diff
@@ -24,11 +25,19 @@ class NextflowSample(BaseModel):
     fastq_forward: List[str]
     fastq_reverse: List[str]
 
+    @validator("fastq_reverse", "fastq_forward")
+    def fastq_not_empty(cls, fastq) -> List[str]:
+        """Verify that fastq files are provided."""
+        if len(fastq) == 0:
+            raise SampleSheetError("Fastq files not provided")
+        return fastq
+
     @validator("fastq_reverse")
-    def fastq_forward_reverse_length_match(cls, fastq_reverse: List[str], values: dict) -> str:
+    def fastq_forward_reverse_length_match(cls, fastq_reverse, values: dict) -> List[str]:
```

Comment:
```suggestion
    def fastq_forward_reverse_length_match(cls, fastqs_reverse: List[str], fastqs_forward: dict) -> List[str]:
```
Not sure about the `values`param I think it could be more descriptive

Reply:
I've seen that's what people use for validators

Reply:
I'll leave it as it is as I cannot come up with anything better